web_widget_datepicker_options/static/src/js/datepicker.js

```diff
@@ -16,38 +16,28 @@
     You should have received a copy of the GNU Affero General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
-/*global openerp, _, $ */
-
-openerp.web_widget_datepicker_options = function (instance) {
 
+odoo.define('inteco.datepicker', function(require) {
```

Comment:
A good rule of thumb is to use `modulename.jsinstancename` in this case would be: `web_widget_datepicker_options.datepicker`

Reply:
Yes. it's done